test/test_nn.py

```diff
@@ -12380,6 +12380,20 @@ def _test_module_empty_input(self, module, inp, check_size=True):
                 self.assertEqual(p.grad, torch.zeros_like(p.grad))
         self.assertEqual(inp.grad, torch.zeros_like(inp))
 
+    def _test_module_empty_inputs(self, module, inputs):
```

Comment:
To avoid the duplication, would it work to change the signature to `_test_module_empty_inputs(self, module, *inputs)` and combine this with `_test_module_empty_input`?

Can call it like `self._test_module_empty_inputs(transformer_decoder, src, target)` for multiple inputs